centreon/packages/js-config/cypress/component/configuration.js

```diff
@@ -45,6 +45,7 @@ module.exports = ({
           if (browser.name === 'chrome' && browser.isHeadless) {
             launchOptions.args.push('--headless=new');
             launchOptions.args.push('--force-color-profile=srgb');
+            launchOptions.args.push('--window-size=1400,1200');
```

Comment:
It breaks snapshots sizes https://github.com/centreon/centreon/actions/runs/11680375631/job/32523274073